### PR TITLE
[Chinese] Fixes and improvements

### PR DESCRIPTION
**Pull request by CikiMomogi**
Impressive work! There are just a few small tweaks I'd suggest making.

### CODE REVIEW

**Comment by roj1512** [comment]
Thanks, @CikiMomogi! I will review this whenever I find time. Can you also revert https://github.com/grammyjs/docs-bot/commit/9bf02432db0cfa2ba6f69f6b7d57fa7a15547e33? (With a PQ.)

**Comment by CikiMomogi** [comment]
No problem

**Comment by CikiMomogi** [comment]
Apparently I don't have a direct access to that repository. I will fork it first.

File: site/docs/zh/guide/introduction.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ code .
 这就是为什么我们希望你知道如何设置你的系统，或者你能够自学。
 （以 _正确的方式_ 安装 Node.js 太复杂了，不适合这个页面。）
 
-如果此时你感到迷茫，你应该抛开 Node.js，转而使用 [Deno](#deno-的前提条件)。
+如果此时你感到迷茫，你应该抛开 Node.js，转而使用 [Deno](#Deno-的前提条件)。
```
Comment (CikiMomogi) [inline site/docs/zh/guide/introduction.md:211]: Making the 'D' uppercase will actually break the link anchor.

**Review by roj1512 — APPROVED** [review]
LGTM. Thank you so much, @CikiMomogi, and the one and only " "!

**Pull request merged**